UK_Power_Generation_Units_BMRS/Python/Code/ukpowergeneration/mapping/README-Mapping-Match-JA-#.md

```diff
@@ -0,0 +1,103 @@
+# README for Mapping-Match-JA-#.py ("#" = version)
+
+## Note
+This is not called by the main functions, but is instead used manually to map, with the output being done ‘Input….csv’ (which must be on this level for the main file ‘Mapping_Match_JA_9.py’ to be run on it). 
+Manually checking the output is reccomended. 
+
+## Description
+This document is a README for:
+ > Mapping-Match-JA-#.py
+
+Three csv files are required. These are not version controlled, but can be found at: 
+> \Dropbox (Cambridge CARES)\CoMo shared\ja685\BMRS\Script-BMRS-API
```

Comment:
Please update to:
`dropbox\CoMo shared\ja685\Modelling-Resources\BMRS\Script-BMRS-API`

Reply:
Done.